README.md

```diff
@@ -119,117 +66,70 @@ After you're finished please remove all the comments and instructions!
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
+> In order to execute the project please follow these steps.
 
 ### Prerequisites
 
```

Comment:
- [ ] OPTIONAL
All main headers from the [ReadMe template](https://github.com/microverseinc/readme-template) should be present in your ReadMe, only the FAQ section is optional.
- [ ]  OPTIONAL
The _Future Features_ section of your ReadMe includes 1-3 potential features to be added later.